### DISCUSSION TITLE
Color Aliases

### DISCUSSION THREAD

**Discussion by itsMarkiS** [PurpurMC/Purpur#579] [__Dismissed__] [answered]
It could have color aliases to use in chat, anvil, or even with plugins
for example, the default minecraft colors
"&1 - &2 - &b - &c - &f"

and also adding a custom alias for the hex colors
example:

"&g" : #00caff
"@n": #5a00ff
"%aa": #1c1651

**Comment by BillyGalbreath** [comment, marked as answer]
Chat is handled via chat plugins.

Purpur already allows colors to be used in books, anvils, and signs.

CraftBukkit allows hex colors to be used in legacy strings with a special &x code. Example: `#123456` is `&x&1&2&3&4&5&6`

**Reply by itsMarkiS** [reply]
yeah but using &x is really annoying... would be cool to let we create placeholders aliases for colors we want..
